playwright/tests/keywordSearch.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { Given, Then, When } from "@cucumber/cucumber";
+import { expect } from "@playwright/test";
+import { CustomWorld } from "../support/setup";
+
+Given(
+  "I am on the DRB home page for searching",
+  async function (this: CustomWorld) {
+    return await this.page.goto(
+      "https://digital-research-books-beta.nypl.org/"
+    );
+  }
+);
+
+When("I input a search term", async function (this: CustomWorld) {
+  await this.page.locator("#searchbar-textinput-search-bar").fill("IBM 1401");
+  await this.page.locator("#searchbar-button-search-bar").click();
```

Comment:
Let's use a locator that isn't a javascript ID here. Maybe `aria-label`?

Reply:
await this.page.locator('[aria-label="Item Search"]').fill("IBM 1401");
await this.page.locator('[data-testid="button"]').click();